apps/platform-app/src/app/[locale]/(profile)/(components)/TogglePublishButton/TogglePublishButton.tsx

```diff
@@ -50,13 +59,15 @@ export const TogglePublishButton = (props: TogglePublishButtonProps) => {
         throw Error('Wrong profile action')
     }
   }
+  
 
   return (
     <div>
-      {showPopup && !loading && (
-        <PublishProfilePopup
-          state={state}
-          onClose={() => setShowPopup(false)}
+      {showPopup && state !== PublishingState.DRAFT && state !== PublishingState.PENDING && (
```

Comment:
**nit:** Why not just `showPopup`?

Reply:
Maybe this was only due to the fact that both the app and db were running locally, but there were situations where I had showPopup as true, while state was DRAFT or PENDING.